config/autoload/doctrine.global.php

```diff
@@ -42,6 +42,9 @@
             'orm_default' => [
                 'auto_generate_proxy_classes' => false
             ]
+        ],
+        'migrations' => [
+            'migrations_directory' => '/docs/generated/doctrine/migrations'
```

Comment:
The specified folder does not currently exist in the repository. 
As discussed in `https://www.doctrine-project.org/projects/doctrine-migrations/en/2.1/reference/configuration.html#migrations-configuration` the folder must be created.